### PR TITLE
Add on tour feature

### PR DESCRIPTION
**Pull request by b5710546232**
Hi Mr.Pawit,

I would like to help you with the navigator feature.
This feature will lead people to each instructor

### CODE REVIEW

File: signaling.html [hunk 1]
```diff
@@ -1,36 +1,44 @@
...
 <html>
+
 <head>
-    <title>WebRTC Signaling Lab</title>
-    <style>
-        p {
-            margin-left: 20px;
-        }
+  <title>WebRTC Signaling Lab</title>
+  <link rel="stylesheet" href="https://unpkg.com/intro.js/minified/introjs.min.css">
```
Comment (pawitp) [inline signaling.html]: Please use versioned URL to prevent breakage in the future.

File: signaling.html [hunk 2]
```diff
@@ -1,36 +1,44 @@
...
+
 <body>
-<h1>WebRTC Labs</h1>
-This page allows you to do WebRTC signalling manually to understand how WebRTC signalling works.
+  <h1>WebRTC Labs</h1>
+  <h2 id="machinetype"></h2>
+  This page allows you to do WebRTC signalling manually to understand how WebRTC signalling works.
+
+  <h2>Tour</h2>
+  Let we guide you <button onclick="intro()">on tour~</button>
```
Comment (pawitp) [inline signaling.html]:
```suggestion
  Let us guide you <button onclick="intro()">on tour~</button>
```

File: signaling.html [hunk 3]
```diff
@@ -44,86 +52,236 @@ <h2>Instruction</h2>
...
+        },
+        ]
+      }).start();
+    }
+    
+    function introReceiver() {
+      introJs().setOptions({
+        steps: [{
+          title: 'Hi, there 👋',
+          intro: `This window is simulated as receiver. We will cal this window as <strong style="color:orange;">Machine B</strong>
```
Comment (pawitp) [inline signaling.html]:
```suggestion
          intro: `This window is simulated as the receiver. We will call this window <strong style="color:orange;">Machine B</strong>
```

**Review by pawitp — CHANGES_REQUESTED** [review]
Nice! Please make minor text changes.

**Comment by b5710546232** [comment]
ありがとう

**Review by pawitp — APPROVED** [review]

**Pull request merged**